tools/tfuser/main.go

```diff
@@ -6,17 +6,27 @@ import (
 	"github.com/threefoldtech/zos/pkg/identity"
 	"github.com/threefoldtech/zos/tools/client"
 
+	"fmt"
 	"os"
 
 	"github.com/urfave/cli"
 )
 
 var (
-	bcdb *client.Client
+	bcdb     *client.Client
+	bcdbAddr string
+	mainSeed string
 )
 
-func main() {
+func requireSeed(c *cli.Context) error {
```

Comment:
This is not needed, you can set the `Required` property of the `cli.StringFlag` to true on line 50 and have the same result.

Reply:
No, this was the initial issue. It you set it as required, you NEED to specify it for ALL subcommands, and you can't specify it for `id` subcommand. This is the exact problem of the main issue :p

Reply:
Right 👌